arcgis-ios-sdk-samples/Features/Time based query/TimeBasedQueryVC.swift

```diff
@@ -0,0 +1,101 @@
+//
+// Copyright 2017 Esri.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//   http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+//
+
+import UIKit
+import ArcGIS
+
+class TimeBasedQueryVC: UIViewController {
+    
+    @IBOutlet weak var mapView: AGSMapView!
+    
+    private var map:AGSMap!
+    
+    private var featureTable:AGSServiceFeatureTable!
+    
+    override func viewDidLoad() {
+        super.viewDidLoad()
+        
+        //initialize map with oceans basemap
+        self.map = AGSMap(basemap: AGSBasemap.oceans())
+        
+        //assign map to the map view
+        self.mapView.map = self.map
+        
+        //create feature table using a url
+        self.featureTable = AGSServiceFeatureTable(url: URL(string: "https://sampleserver6.arcgisonline.com/arcgis/rest/services/Hurricanes/MapServer/0")!)
+        
+        //define the request mode
+        self.featureTable.featureRequestMode = .manualCache
+        
+        //load the feature table
+        self.featureTable.load { [weak self] (error) in
```

Comment:
on second thoughts, lets simplify this by removing the code to load the feature table
We can just directly create layer from table and add it to map. 
populate() will implicitly load the table and if there are errors it'll get printed in the completion block.